### PR TITLE
Add tests & docs for new playbook endpoint

### PR DESCRIPTION
**Pull request by biancarosa**

### CODE REVIEW

File: cmd/spec/openapi.yaml [hunk 1]
```diff
@@ -620,6 +618,35 @@ paths:
...
       summary: Executes a device update.
       description: Executes a device update.
+  /updates/{updateID}/update-playbook.yml:
+    get:
+      operationId: GetUpdatePlaybook
+      parameters:
+        - name: updateID
+          in: path
+          required: true
+          description: An unique ID to identify the update that you want to retrieve your playbook for
```
Comment (AaronH88) [inline cmd/spec/openapi.yaml:628]: "An unique ID to identify the update playbook" ?

File: cmd/spec/path.yaml [hunk 2]
```diff
@@ -220,6 +220,35 @@ paths:
...
       summary: Executes a device update.
       description: Executes a device update.
+  /updates/{updateID}/update-playbook.yml:
+    get:
+      operationId: GetUpdatePlaybook
+      parameters:
+        - name: updateID
+          in: path
+          required: true
+          description: An unique ID to identify the update that you want to retrieve your playbook for
```
Comment (AaronH88) [inline cmd/spec/path.yaml:230]: Same as other file: "An unique ID to identify the update playbook" ?

**Review by AaronH88 — COMMENTED** [review]
Small re-wording suggestion.

Other than that, looks great! Ok to merge with or without the suggestion.

**Review by acosferreira — APPROVED** [review]
LGTM

**Pull request merged**